src/test/java/hudson/scm/SubversionSCMTest.java

```diff
@@ -1223,8 +1223,12 @@ public void invalidTest3936()  throws Exception {
 
     private void attemptAccess(ISVNAuthenticationManager m) throws SVNException {
         SVNRepository repository = SVNRepositoryFactory.create(repo);
-        repository.setAuthenticationManager(m);
-        repository.testConnection();
+        try {
+            repository.setAuthenticationManager(m);
+            repository.testConnection();
+        } finally {
+            repository.closeSession();
+        }
```

Comment:
Not needed to fix JENKINS-42186 but is leaking HTTP connections during test runs.